industrial_robot_client/src/joint_trajectory_action.cpp

```diff
@@ -45,12 +45,28 @@ const double JointTrajectoryAction::DEFAULT_GOAL_THRESHOLD_ = 0.01;
 JointTrajectoryAction::JointTrajectoryAction() :
     action_server_(node_, "joint_trajectory_action", boost::bind(&JointTrajectoryAction::goalCB, this, _1),
                    boost::bind(&JointTrajectoryAction::cancelCB, this, _1), false), has_active_goal_(false),
-                   controller_alive_(false), has_moved_once_(false), name_("joint_trajectory_action")
+                   controller_alive_(false), has_moved_once_(false), name_("joint_trajectory_action"),
+                   ignore_motion_server_error_(false), consider_status_unknowns_ok_(false)
 {
   ros::NodeHandle pn("~");
 
   pn.param("constraints/goal_threshold", goal_threshold_, DEFAULT_GOAL_THRESHOLD_);
 
+  // Two parameters for bw-compatibility with the 'old' behaviour.
+  // Setting these by default to TRUE, to maintain the previous behaviour
+  pn.param("ignore_motion_server_error", ignore_motion_server_error_, true);
```

Comment:
```suggestion
  pn.param("ignore_motion_server_error", ignore_motion_server_error_, ignore_motion_server_error_);
```
again a mater of preference I guess but it reduces the number of places where you can change the default